tests/FixerFactoryTest.php

```diff
@@ -415,7 +415,10 @@ public function testFixerDefinitions(FixerInterface $fixer)
             $this->assertStringIsNotEmpty($definition->getConfigurationDescription(), sprintf('[%s] Configuration description is required.', $fixer->getName()));
             $default = $definition->getDefaultConfiguration();
             $this->assertInternalType('array', $default, sprintf('[%s] Default configuration must be an array.', $fixer->getName()));
-            $this->assertNotEmpty('array', $default, sprintf('[%s] Default configuration is required.', $fixer->getName()));
+
+            if (!in_array($fixer->getName(), array('general_phpdoc_annotation_remove', 'psr0'), true)) {
+                $this->assertNotEmpty($default, sprintf('[%s] Default configuration is required.', $fixer->getName()));
```

Comment:
i don't wanna focus now on how to handle those exceptions better , but we were calling assertion wrongly